Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php

```diff
@@ -106,7 +111,6 @@ public static function cleanUpSecurity()
 
     /**
      * @Given /^I am not authenticated$/
-     * @throws \Neos\Flow\ObjectManagement\Exception\UnknownObjectException
```

Comment:
Why did you remove all those `@throws`  annotations?

Reply:
Probably ide auto stuff since they are not thrown in that method but somwhere deeper?